# Camera intrinsics  
  # mid360
  fx: 522.123514287681
  fy: 522.275153384482
  cx: 773.466430504725
  cy: 534.053165700174
  k1: 0.00324949759262203
  k2: -0.0171040538369167
  p1: 0.000669657443377146
  p2: -0.000350205468789575

  # avia
  # fx: 1215.31801774424
  # fy: 1214.72961288138
  # cx: 1047.86571859677
  # cy: 745.068353101898
  # k1: -0.33574781188503
  # k2: 0.10996870793601
  # p1: 0.000157303079833973
  # p2: 0.000544930726278493

  # ouster
  # fx: 1466.16500795563
  # fy: 1465.32657394431
  # cx: 1214.39964380518
  # cy: 1010.97681425650
  # k1: -0.0458344972140773
  # k2: 0.0896961240772692
  # p1: -0.000731746350601389
  # p2: -0.000201478932371979

# Calibration target parameters
  marker_size: 0.20 # ArUco marker size (our test data uses 0.16m; adjust to match your marker size)
  delta_width_qr_center: 0.55 # Half the distance between the centers of two markers in the horizontal direction 
  delta_height_qr_center: 0.35 # Half the distance between the centers of two markers in the vertical direction
  delta_width_circles: 0.5 # Distance between the centers of two circles in the horizontal direction
  delta_height_circles: 0.4 # Distance between the centers of two circles in the vertical direction
  circle_radius: 0.12 # Radius of the circle 

# Distance filter
  # Leave Rcl and Pcl empty if you do not want the filter parameters to be estimated automatically (DO NOT DELETE THEM)
  Rcl: []
  Pcl: []
  x_min: 1.5
  x_max: 3.0
  y_min: -1.5
  y_max: 2.0
  z_min: -0.5 # avia and mid360: -0.5 ouster: -0.8
  z_max: 2.0

# Input
  lidar_topic: "/livox/lidar" # /ouster/points /livox/lidar
  bag_path: "$(find fast_calib)/calib_data/mid360/11.bag"
  image_path: "$(find fast_calib)/calib_data/mid360/11.png"

# Output
  output_path: "$(find fast_calib)/output"